src/main/java/com/vonage/quickstart/verify/StartVerificationWithWorkflow.java

```diff
@@ -37,10 +37,10 @@ public static void main(String[] args) {
         String VONAGE_API_SECRET = envVar("VONAGE_API_SECRET");
 
         String RECIPIENT_NUMBER = envVar("RECIPIENT_NUMBER");
-        String VONAGE_BRAND = envVar("BRAND_NAME");
+        String BRAND_NAME = envVar("BRAND_NAME");
```

Comment:
I'd flag this as another spec change, because why do we need `BRAND_NAME` and `VONAGE_BRAND_NAME`?

Reply:
IMO `BRAND_NAME` is correct here. it's the brand that the verify API is going to announce the request is from.